apps/els_lsp/src/els_log_notification.erl

```diff
@@ -0,0 +1,50 @@
+-module(els_log_notification).
+
+-export([start/0]).
+-export([log/2]).
+
+-type lsp_message_type() :: 1 | 2 | 3 | 4.
+
+-spec start() -> ignore.
+start() ->
+    {ok, LoggingLevel} = application:get_env(els_core, log_level),
+    ok = logger:add_handler( ?MODULE
+                           , ?MODULE
+                           , #{ level => LoggingLevel }
+                           ),
+    ignore.
+
+-spec log(logger:log_event(), logger:config_handler()) -> ok.
+log(#{level := Level} = LogEvent, _Config) ->
+    try
+        Msg = logger_formatter:format(LogEvent, #{ template => [ "[", time, "] "
```

Comment:
Something that we should also consider for the other logs is to make these messages a bit more readable. For example, we could start with `[level] msg` and have the remaining parts at the end. Not sure if it's easy, but we could only have the filename instead of the full path. WDYT?

Reply:
agreed, let me play with the format a bit